survey/management/commands/importscenes.py

```diff
@@ -25,21 +24,18 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
-        Scene.objects.all().delete()
```

Comment:
Lese ich das richtig, dass man jetzt Szenen nicht mehr entfernen kann, indem man eine neue Szenenliste importiert, in der diese nicht mehr auftauchen?